gulpfile.js

```diff
@@ -464,12 +466,19 @@ function deployCesium(bucketName, uploadDirectory, cacheControl, done) {
             }
 
             console.log('Cleaning up old files...');
+
+            var objectToDelete = [];
+            existingBlobs.forEach(function(file) {
+                //Don't delete generate zip files.
+                if (!/\.(zip|tgz)$/) {
```

Comment:
The delete request failed when running in travis. I think this regex should be calling `test` on  `file`.